kore/src/Kore/Repl/Data.hs

```diff
@@ -303,7 +314,14 @@ helpText =
     \\n\
     \(*) If an edge is labeled as Simpl/RD it means that\
     \ either the target node was reached using the SMT solver\
-    \ or it was reached through the Remove Destination step."
+    \ or it was reached through the Remove Destination step.\n\
+    \\n\
+    \Rule names can be added in two ways:\n\
+    \    a) rule <k> ... </k> [label(myName)]\n\
+    \    b) rule [myName] : <k> ... </k>\n\
+    \Names added via a) can be used as-is.\n\
+    \Namess added via b) need to be prefixed with the module name followed by dot,\
+    \ e.g. IMP.myName"
```

Comment:
Might be worth noting that names declared via a) that match our indexing syntax will not work (e.g. if they name their rule `a5`).